src/Microsoft.Framework.Runtime/ExportProviders/Resources/CreateCSharpManifestResourceName.cs

```diff
@@ -51,110 +52,112 @@ public static string CreateManifestName(string fileName, string rootNamespace)
         // The code below the same is same as here: https://raw.githubusercontent.com/Microsoft/msbuild/41b137cd8805079af7792995e044521d62fcb005/src/XMakeTasks/CreateManifestResourceName.cs
 
         /// <summary>
-        /// This method is provided for compatibility with Everett which used to convert parts of resource names into
+        /// This method is provided for compatibility with MsBuild which used to convert parts of resource names into
         /// valid identifiers
         /// </summary>
-        /// <param name="name"></param>
-        /// <returns></returns>
-        private static string MakeValidEverettIdentifier(string name)
+        private static string MakeValidIdentifier(string name)
         {
-            StringBuilder everettId = new StringBuilder(name.Length);
+            var id = new StringBuilder(name.Length);
 
             // split the name into folder names
-            string[] subNames = name.Split(new char[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar });
+            var subNames = name.Split(new char[] { Path.DirectorySeparatorChar, Path.AltDirectorySeparatorChar });
```

Comment:
Though our love for the _var_ is clear, I think using the `string[]` us more appropriate here since the right hand express doesn't clearly indicate the return value type.